src/Google.Pubsub.V1/SubscriberClient.cs

```diff
@@ -128,73 +111,39 @@ public static class ServiceDefaults
         /// <summary>
-        /// Create a <see cref="SubscriberClient"/> from default credentials.
+        /// Asynchronously creates a <see cref="SubscriberClient"/>, applying defaults for all unspecified settings.
         /// </summary>
-        /// <param name="settings">Optional <see cref="SubscriberSettings"/>.</param>
-        /// <param name="serviceEndpointSettings">Optional <see cref="ServiceEndpointSettings"/>.</param>
-        /// <param name="credentialScopes">Optional scopes for default credentials.</param>
-        /// <returns>A newly created <see cref="SubscriberClient"/>.</returns>
-        public static SubscriberClient CreateFromDefaultCredentials(
+        /// <param name="endpoint">Optional <see cref="ServiceEndpoint"/>.</param>
+        /// <param name="settings">Optional <see cref="PublisherSettings"/>.</param>
+        /// <param name="credentials">Optional <see cref="ChannelCredentials"/>.</param>
+        /// <returns></returns>
+        public static async Task<SubscriberClient> CreateAsync(
+            ServiceEndpoint endpoint = null,
             SubscriberSettings settings = null,
-            ServiceEndpointSettings serviceEndpointSettings = null,
-            IEnumerable<string> credentialScopes = null)
+            ChannelCredentials credentials = null)
         {
-            return ClientHelper.CreateFromDefaultCredentials(
-                settings, serviceEndpointSettings, credentialScopes, ServiceDefaults.Scopes, CreateFromCredentials);
+            var channel = await ClientHelper.CreateChannelAsync(endpoint ?? DefaultEndpoint, credentials).ConfigureAwait(false);
+            var grpcClient = new Subscriber.SubscriberClient(channel);
+            return new SubscriberClientImpl(grpcClient, settings);
         }
 
         /// <summary>
-        /// Create a <see cref="SubscriberClient"/> from the specified credentials.
+        /// Synchronously creates a <see cref="SubscriberClient"/>, applying defaults for all unspecified settings.
         /// </summary>
-        /// <param name="credentials">The credentials with which to configure the GRPC channel.</param>
-        /// <param name="settings">Optional <see cref="SubscriberSettings"/>.</param>
-        /// <param name="serviceEndpointSettings">Optional <see cref="ServiceEndpointSettings"/>.</param>
+        /// <param name="endpoint">Optional <see cref="ServiceEndpoint"/>.</param>
+        /// <param name="settings">Optional <see cref="PublisherSettings"/>.</param>
+        /// <param name="credentials">Optional <see cref="ChannelCredentials"/>.</param>
         /// <returns></returns>
-        public static SubscriberClient CreateFromCredentials(
-            ChannelCredentials credentials,
+        public static SubscriberClient Create(
+            ServiceEndpoint endpoint = null,
             SubscriberSettings settings = null,
-            ServiceEndpointSettings serviceEndpointSettings = null)
+            ChannelCredentials credentials = null)
```

Comment:
The point is that most of the time - as you said - applications just want to use the application default parameters. So they use

```csharp
var client = SubscriberClient.Create();
```

... and they're talking to the default endpoint, using the application default credentials. Job done.